### PR TITLE
[HCBS] QMS - Set CMIT Number & Add QM to LTSS-5

### PR DESCRIPTION
**Pull request by rocio-desantiago**
### Description
This is the fix to a trailing TO-DO based on [Routing By Page Id](https://github.com/Enterprise-CMCS/macpro-mdct-hcbs/pull/54) and [setting up](https://github.com/Enterprise-CMCS/macpro-mdct-hcbs/pull/66) the measures, AND including the ONE and ONLY quality measure (MLTSS) for LTSS-5 at the bottom of the page. 

Prior to this fix: cmit wasn't being set, therefore the Quality Measures was showing up empty even though previously we did have those showing up.
emptyQualityMeasures

### Related ticket(s)

CMDCT-4147 (kinda)

---
### How to test

**Cloudfront URL: https://d35uyj0spo417i.cloudfront.net/**
1. Sign into HCBS, any state user
2. Create a QM report
3. On the sidebar menu, select [Optional Measure Results]. 
   - The 6th row down will be LTSS-5, CMIT #1255.
4. Click [Edit] and you be taken to the LTSS-5 measure overview page.
5. Check that the questions matches what is written in the jira ticket: [CMDCT-4147](https://jiraent.cms.gov/browse/CMDCT-4147) - Less questions than other LTSS measures
6. You should see a page that looks like this
    ltss-5-cmit
7. Additionally confirm that quality measures are showing up for those as well as show in screenshot of LTSS-3 below:
  - [Required Measure Results] LTSS1, LTSS-2, LTSS-6, LTSS-7, LTSS-8 & 
  - [Optional Measure Results] LTSS-3, LTSS-4, LTSS-5
     LTSS-3

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

### CODE REVIEW

**Review by ajaitasaini — APPROVED** [review]

File: services/ui-src/src/components/report/MeasureTable.tsx [hunk 1]
```diff
@@ -50,11 +45,10 @@ export const MeasureTableElement = (props: PageElementProps) => {
   const { reportType, state, reportId } = useParams();
   const navigate = useNavigate();
 
-  // TO-DO: Where does cmit need to be set? Does it happen onEdit click?
-  const handleEditClick = (measureId: string) => {
+  const handleEditClick = (measureId: string, cmit: number | undefined) => {
```
[the remark this answers is not in the document]
Comment (rocio-desantiago) [reply services/ui-src/src/components/report/MeasureTable.tsx:48]: That's a great question. I just copied from `buildModal`. @ailZhou a penny for your thoughts?

[the remark this answers is not in the document]
Comment (ailZhou) [reply services/ui-src/src/components/report/MeasureTable.tsx:48]: i don't think it ever is undefined but tagging @BearHanded in case i'm wrong

[the remark this answers is not in the document]
Comment (BearHanded) [reply services/ui-src/src/components/report/MeasureTable.tsx:48]: I think we have some flexibility here. Cameron seemed receptive to our judgement. A CMIT may be undeclared while we are developing a measure, if the measure hasn't been finalized by CMS. Once they finalize it, it gets a number. We can treat it as undefined, or more likely, give it a temp unique number we can modify later (ex. 9999). That is what we are doing for one of our measures at the moment.

In short, it should not be undefined, but it's an area of some weirdness. 

I wouldn't be surprised if in the future we found out we have to collect something that doesn't have a CMIT, and need to reevaluate.

**Review by ailZhou — APPROVED** [review]

File: services/ui-src/src/components/report/MeasureTable.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rocio-desantiago) [reply services/ui-src/src/components/report/MeasureTable.tsx:48]: Cool. I'll address in other PR 😄
